specification/dimensions/commitmentid.md

```diff
@@ -0,0 +1,32 @@
+# Commitment ID
+
+A commitment is a contractual commitment to use a minimum amount of spending on resources for a fixed term in exchange for discounts on those resources. Commitments can cover various resources such as virtual machines, or databases. A provider can offer various pricing models, payment options, and term durations for its commitment offerings.
```

Comment:
A few thoughts:
1. Thinking out loud: Should we say "A commitment-based discount is..."? There are other commitments that aren't "commitment-based discounts". Not sure if we want to generalize this to include those or not.
2. Does "minimum" make it sound like they're only valuable if you have 100% utilization?
3. We should work in a reference to differentiate usage-based and cost-based commitments. (Added something, but I feel like it could be better.)
```suggestion
A commitment-based discount is a contractual commitment to use a certain amount of a specific type of usage for a fixed term in exchange for a discounted unit price. Commitment-based discounts can cover either usage or cost of various resources such as virtual machines or databases. A provider can offer various pricing models, payment options, and term durations for commitment-based discounts.
```